README.md

```diff
@@ -63,3 +244,7 @@ Facilitator can be started in below two ways :-
     * Replace `<gateway-config>` with the path to gateway-config.json.	    
     
   * **Note** : Both `--mosaic-config` and `--gateway-config` are together not allowed in command.
+
+
+    
+
```

Comment:
We haven't added a section in here to help users to just move tokens around using facilitators hosted by others.